docs/cli/dotnet-restore.md

```diff
@@ -0,0 +1,77 @@
+dotnet-restore
+==============
+
+## NAME
+
+`dotnet-restore` - restores the dependencies and tools of a project
+
+## SYNOPSIS
+
+dotnet-restore [--source]  
+    [--packages] [--disable-parallel]  
+    [--fallbacksource] [--configfile] [--verbosity]
+    [< root >]  
+
+## DESCRIPTION
+
+`dotnet-restore` will use NuGet to restore dependencies as well as project-specific tools that are specified in the 
+project.json file. By default, the restoration of dependencies and tools will be done in parallel.
+
+In order to restore the dependencies, NuGet needs feeds where the packages are located. Feeds are usually provided via the 
+NuGet.config configuration file; a default one is present when CLI tools are installed. You can specify more feeds by 
+creating your own NuGet.config file in the project directory. Feeds can also be specified per invocation on the command line. 
+
+For dependencies, you can specify where the restored packages will be placed during the restore operation using the 
+`--packages` argument. 
```

Comment:
Perhaps state here where the default Nuget cache is on each platform, as has been done in the other docs. (IE, install script calls out clearly that things go to %localappdata% or ~.